experimental/GModule/src/Cohomology.jl

```diff
@@ -453,6 +490,13 @@ function induce(C::GModule{<:Oscar.GAPGroup}, h::Map, D = nothing, mDC = nothing
   return iC, h
 end
 
+function is_induced(C::GModule)
+  if hasfield(typeof(indC), :__attrs)
```

Comment:
there is no `indC` defined here. Do you mean `C`?

And of course we shouldn't check for `__attrs` at all here, I'll try to suggest something better later